plugins/rapture/nexus-coreui-plugin/src/main/java/org/sonatype/nexus/coreui/RepositoryComponent.groovy

```diff
@@ -108,13 +127,30 @@ extends DirectComponentSupport
         name: input.name,
         type: input.type,
         format: input.format,
+        recipe: input.configuration.recipeName,
         online: input.facet(ViewFacet).online,
         status: buildStatus(input),
-        attributes: attributeConverter.asAttributes(input.configuration.attributes),
+        attributes: input.configuration.attributes,
         url: "${BaseUrlHolder.get()}/repository/${input.name}"
     )
   }
 
+  /**
```

Comment:
http://stackoverflow.com/questions/15507997/how-to-prevent-gson-from-expressing-integers-as-floats

I tried adding a Double converter type but it didn't appear to have any effect so I need to dig further into ExtDirect/GSON to see what's going on. That approach is universal across all requests - so no double data in any Map<String, Object> - and sounds like it would require upgrading the lib to get around a known issue with converters for primitive types.
This works, but is an obvious hack :(